src/components/KeyValueTable/KeyValueTable.tsx

```diff
@@ -0,0 +1,37 @@
+import { Grid, GridItem } from '@patternfly/react-core';
+
+import styles from './KeyValueTable.module.css';
+
+interface IKeyValueEntryProps {
+  name: React.ReactNode;
+  value: React.ReactNode;
+}
+
+const KeyValueEntry = ({ name, value }: IKeyValueEntryProps) => (
+  <>
+    <GridItem md={3} sm={12} className={styles['key']}>
+      {name}
+    </GridItem>
+    <GridItem md={9} sm={12} className={value ? '' : styles['value']}>
+      {value ?? 'Empty'}
+    </GridItem>
+  </>
+);
+
+interface IKeyValueTableProps {
+  keyValueArray: {
+    key: React.ReactNode;
```

Comment:
let's use `name` to be consistent with the component above + to avoid `key` conflicts ( https://reactjs.org/docs/lists-and-keys.html )